_dashboards/management/advanced-settings.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: default
+title: Advanced settings
+parent: Dashboards Management
```

Comment:
@joshuarrrr Should we link to the actual ISO source instead of Wikipedia? (https://www.iso.org/iso-8601-date-and-time-format.html instead of https://en.wikipedia.org/wiki/ISO_8601#Time_intervals)